e2e-tests/mongod-major-upgrade-sharded/run

```diff
@@ -104,16 +86,10 @@ function main() {
 
 	target_generation=2
 	for version in ${versions_to_verify[@]}; do
-
-		backup_name_minio="backup-minio-${target_generation}"
-		run_backup minio ${backup_name_minio}
-		wait_backup ${backup_name_minio}
-
 		kubectl_bin patch psmdb/${cluster%%-rs0} \
 			--type=json \
 			-p='[
 					{"op":"replace","path":"/spec/upgradeOptions/apply","value": "'${version}'-recommended"},
-					{"op":"replace","path":"/spec/upgradeOptions/setFCV","value": true}
 				]'
```

Comment:
Why did we split these commands?

Reply:
[source](https://www.mongodb.com/docs/v5.0/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion)

It is recommended that after upgrading, you allow your deployment to run without enabling these features for a burn-in period to ensure the likelihood of downgrade is minimal. When you are confident that the likelihood of downgrade is minimal, enable these features.